src/main/java/com/digitalasset/testing/junit4/Sandbox.java

```diff
@@ -110,19 +115,35 @@ public SandboxBuilder logLevel(LogLevel logLevel) {
       return this;
     }
 
+    public SandboxBuilder projectRoot(Path projectRoot) {
+      try {
+        if (Files.list(projectRoot).noneMatch(damlYamlP()))
+          throw new IllegalArgumentException("Project root must contain a daml.yaml");
```

Comment:
The assistant will search for the daml.yaml by itself (recursively in parent dir.s) when invoked. A working directory should not have to contain the file IMHO.
Alternatively, point to the file itself.